vizro-core/src/vizro/models/_components/graph.py

```diff
@@ -70,33 +71,49 @@ def __getitem__(self, arg_name: str):
             return self.type
         return self.figure[arg_name]
 
+    @_log_call
+    def pre_build(self):
```

Comment:
Is it easy to do this in a validator for `title` rather than `pre_build`? If so then I think we should prefer that.